shell/artifacts/Common/source/ShowCollection.js

```diff
@@ -8,10 +8,10 @@
 
 'use strict';
 
-defineParticle(({DomParticle, resolver}) => {
+defineParticle(({DomParticle, html, resolver}) => {
```

Comment:
Double checking that you want this in here as well.

Reply:
Sure, it's employed on L52. It makes the templates so much easier to read on supported editors.